UI/MainMenu/MainMenu.gd

```diff
@@ -127,61 +353,155 @@ func _on_HTTPRequest_request_completed(result, _response_code, _headers, body):
 		return
 	gutHubReleaseLabel.text = "Latest github release: Nothing found"
 
-func _on_GithubReleasesButton_pressed():
-	var _ok = OS.shell_open("https://github.com/Alexofp/BDCC/releases")
-
+func prepareToGame():
+	showGameScreen()
+	change_content_on_left_panel(1)
+	change_content_on_right_panel(1)
+	
+func returnToTitle():
+	showTitle()
+	hidePad(false)
+	change_content_on_left_panel(0)
+	change_content_on_right_panel(0)
+
+func removeCharacterFromPanel(id):
+	NPC_art_panel.removeCharacter(id)
+
+func hidePad(vis : bool):
+	pad.visible = !vis
+
+#Covers your pad under glass to prevent clicking)))
+func blockPad(yes = true):
```

Comment:
what is pad and why do we need to prevent clicking on it :o

Reply:
Unused, but funny feature what come trough developing GUI)